src/warc2zim/converter.py

```diff
@@ -127,6 +127,20 @@ def __init__(self, args):
         with tempfile.NamedTemporaryFile(dir=self.output, delete=True) as fh:
             logger.debug(f"Confirming output is writable using {fh.name}")
 
+        # ensure zim file is creatable with the given name
```

Comment:
Idem

Reply:
Fixed in [00823a9](https://github.com/openzim/warc2zim/pull/234/commits/00823a9a4c0cfe42bb3b75c6d7ed450519933439)